rules/linux/process_creation/proc_creation_lnx_browser_information_discovery.yaml

```diff
@@ -0,0 +1,37 @@
+title: Browser Information Discovery - Linux
+id: 53f9703e-abc5-4eff-945e-2ab75c206c63
+status: experimental
+description: Adversaries may enumerate information about browsers to learn more about compromised environments.Data saved by browsers (such as bookmarks, accounts, and browsing history) may reveal a variety of personal information about users (e.g., banking sites, relationships/interests, social media) and details about internal network resources like servers, tools, dashboards, or other infrastructure.
```

Comment:
Description must state what the detection is doing before talking how adversaries can abuse this. They all start with Detects.
See other rules in the repo